termite.cc

```diff
@@ -44,6 +44,21 @@
 
 #include "url_regex.hh"
 
+typedef struct {
+  const char *pattern;
+} TerminalRegexPattern;
+
+static const TerminalRegexPattern url_regex_patterns[] = {
```

Comment:
Any reason why this cannot be an array of `const char*`? Something like this:

```suggestion
static const char *const url_regex_patterns[] = {
```

Reply:
no valid reason other than my lack of competence in cpp